pmd-lang-test/pom.xml

```diff
@@ -13,7 +13,6 @@
         <groupId>net.sourceforge.pmd</groupId>
         <artifactId>pmd</artifactId>
         <version>6.30.0-SNAPSHOT</version>
-        <relativePath>../</relativePath>
```

Comment:
I think, we should keep this:

> Notice the relativePath element. It is not required, but may be used as a signifier to Maven to first search the path given for this project's parent, before searching the local and then remote repositories.
> https://maven.apache.org/pom.html#Inheritance

Reply:
Ah ok, I didn't know it would be looking into external repositories